doc/rbd/rbd-mirroring.rst

```diff
@@ -231,6 +231,55 @@ pool as follows:
    same name exists on the destination cluster, that pool will be used.
 #. If neither of the above is true, no data pool will be set.
 
+Namespace Configuration
+=======================
+
+Mirroring can be configured for a namespace in a pool. The pool must already
+have been configured for mirroring. The namespace can be mirrored to a namespace
+with the same or a different name in the pool.
+
+Enable Mirroring
+----------------
+
+To enable mirroring on a namespace with ``rbd``, issue the ``mirror pool enable``
+subcommand with the namespace spec and the mirroring mode, and an optional
+remote namespace name::
+
+        rbd mirror pool enable {pool-name}/{local-namespace} {mode} --remote-namespace {namespace-name}
+
+The mirroring mode can either be ``image`` or ``pool``:
+
+* **image**: When configured in ``image`` mode, mirroring must
+  `explicitly enabled`_ on each image.
+* **pool** (default):  When configured in ``pool`` mode, all images in the namespace
+  with the journaling feature enabled are mirrored.
+
+For example::
+
+        $ rbd --cluster site-a mirror pool enable image-pool/namespace-a image --remote-namespace namespace-b
+        $ rbd --cluster site-b mirror pool enable image-pool/namespace-b image --remote-namespace namespace-a
+
+This will set up image mode mirroring between image-pool/namespace-a on cluster
+site-a and image-pool/namespace-b on cluster site-b.
+If the ``--remote-namespace`` option is not provided, the namespace will be
+mirrored to the same namespace name on the remote cluster.
```

Comment:
Suggest a tweak since there can be multiple `namespace-a` namespaces in the remote cluster (in different pools):
```suggestion
mirrored to a namespace with the same name in the remote pool.
```